web/src/people.ts

```diff
@@ -1458,8 +1458,15 @@ export function add_active_user(person: User): void {
     non_active_user_dict.delete(person.user_id);
 }
 
-export const is_person_active = (user_id: number): boolean => {
+export const is_person_active = (
+    user_id: number,
+    allow_missing_user = !settings_data.user_can_access_all_other_users(),
```

Comment:
This will mean that this will return `true` even when `user_id` is invalid, i.e. there is no user at all with such ID. But this is the case with other functions like `get_user_by_id_assert_valid` as well. And it is fine as we expect only valid user IDs to be passed to this function.